docker-gen.go

```diff
@@ -72,6 +72,7 @@ type RuntimeContainer struct {
 	Volumes      map[string]Volume
 	Node         SwarmNode
 	Labels       map[string]string
+	Server       ServerInfo
```

Comment:
Seems like this should go on the `Context` and not on each `RuntimeContainer`.  That way you could access it directly without a reference to a container.  

See: https://github.com/jwilder/docker-gen/blob/master/docker-gen.go#L131